src/sphinx/formats/jdkpackager.rst

```diff
@@ -136,7 +152,19 @@ Here's what the build file looks like:
 
     jdkPackagerType := "installer"
 
+    jdkPackagerJVMArgs := Seq("-Xmx1g")
+
+    jdkPackagerProperties := Map("app.name" -> name.value, "app.version" -> version.value)
 
+    jdkPackagerAppArgs := Seq(maintainer.value, packageSummary.value, packageDescription.value)
+
+    jdkPackagerAssociations := Seq(
+        FileAssociation("foobar", "application/foobar", "Foobar file type"),
+        FileAssociation("barbaz", "application/barbaz", "Barbaz file type", jdkAppIcon.value)
+    )
 
```

Comment:
Could you add one small section here, showing or to add other jdk-locations (if not found by default), like this

```scala
jdkPackagerTool := jdkPackagerTool.value orElse Some(file("/usr/lib/jvm/java-8-oracle/bin/javapackager"))
```

and how setting `JDK_HOME` or `JAVA_HOME` may help ;)